force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.html

```diff
@@ -143,38 +143,50 @@
                                     ></lightning-spinner>
                                 </div>
                             </template>
-                            <template if:true={showSavedIcon} role="alert">
-                                <lightning-icon
-                                    tabindex="0"
-                                    icon-name="utility:success"
-                                    alternative-text={labels.saved}
-                                    size="x-small"
-                                    variant="success"
-                                    title={labels.saved}
-                                ></lightning-icon>
-                                <span class="slds-var-p-left_small">{labels.saved}</span>
+                            <template if:true={showSavedIcon}>
+                                <div role="alert">
```

Comment:
@ptewson-sfdo, Can we remove the alert from this div here since the screen reader is reading these out too instead of just reading the toast.

Reply:
Sure.. I can ask the team

Reply:
@ptewson-sfdo, Tarren agrees that we should remove this so the screen reader can only announce the toast and not the individual row saved or error label.